src/App.js

```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] For this task, the `App.js` should be class-based, not functional-based. Let's try to update it.
See example:
```js
class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```

Reply:
I have done this but linter for App.js forced me to make it a pure function!